File: Classwork/RandomNumber.py

guess_number = 20
input_number = int(input("Guess a number: "))

while input_number != guess_number:
    if input_number > guess_number:
        print("Too high")
    elif input_number < guess_number:
        print("Too low")
    elif input_number == guess_number:
        print("you are correct ", guess_number, "is the correct number")
    input_number = int(input("Guess a number: "))

File: Classwork/RandomNumber.java
import java.util.Scanner;

public class RandomNumber{
public static void main(String[] args){
 Scanner input = new Scanner(System.in);
 int fixed_value = 20;

 System.out.println("Enter a number: ");
  int input_number = input.nextInt();
  
 while(input_number != fixed_value){
  if(input_number > fixed_value){
  System.out.println("Too high");
  }else if(input_number < fixed_value){
  System.out.println("Too low");
 }else if(input_number == fixed_value){
  System.out.println("you are correct" + fixed_value + "is the correct number");
 }
 System.out.println("Enter a number: ");
   input_number = input.nextInt();
  }
  


}

}
